packages/assets-controllers/src/TokenListController.ts

```diff
@@ -237,7 +237,7 @@ export class TokenListController extends PollingController<
    * @param networkClientId - The ID of the network client triggering the fetch.
    * @returns A promise that resolves when this operation completes.
```

Comment:
Since we can't use JavaScript to mark this as private, do we want to add a `@private` tag to this JSDoc block as another form of communication?

Reply:
done here: https://github.com/MetaMask/core/pull/1810/commits/8536555a04fe512dfdffdf93875b4c56cd22dfd4